numpy/core/src/multiarray/methods.c

```diff
@@ -3076,9 +3082,11 @@ NPY_NO_EXPORT PyMethodDef array_methods[] = {
     {"__dlpack__",
         (PyCFunction)array_dlpack,
         METH_FASTCALL | METH_KEYWORDS, NULL},
-
     {"__dlpack_device__",
         (PyCFunction)array_dlpack_device,
         METH_NOARGS, NULL},
+    {"bitwise_count",
+        (PyCFunction)array_bitwise_count,
+        METH_VARARGS | METH_KEYWORDS, NULL},
```

Comment:
This might be handy? If we want to support it on other sequences? I'm guessing there will be some other way though without this addition? happy to do both

Reply:
I removed it via [d7f364f](https://github.com/numpy/numpy/pull/21429/commits/d7f364fee9d1a85b08126ff723f8d884f801e8ad). I'm a bit confused now, I thought this adds `bitwise_count` to np namespace?